### ISSUE TITLE
[bug] jwt-util.ts type issue

### ISSUE THREAD

**Issue by chobkyu** [chobkyu/hansig-server#46]
![image](https://github.com/chobkyu/hansig-server/assets/98572357/1b403d41-2a9a-4b7e-9500-540cd2cd728b)

현재 위 에러가 나고 있는데 준영님 개발 할 땐 위 에러가 나지 않았나요???

@jeon0711

**Comment by jeon0711** [comment]
동일한 경고 메세지는 뜹니다만 진행이 상관없이 되어서요. 실행이 안되거나 문제가 생기지는 않았습니다. error객체가 message속성를 항상 포함하고 있으니까 실제 문제는 없으리라 생각했습니다. 일단 수정하겠습니다.

**Comment by jeon0711** [comment]
location branch에서 Error의 인스턴스인지 확인하도록 수정하겠습니다.

**Comment by chobkyu** [comment]
네 저런 타입 이슈는 환경마다 다르긴 한데 실행이 안될 수도 있어서요. 일단 저는 any로 놓고 다른 작업 해놓긴 했습니다
